pkg/logging/field.go

```diff
@@ -2,11 +2,19 @@ package logging
 
 import "go.uber.org/zap"
 
+type DataField interface {
+	GetField() zap.Field
```

Comment:
Would this be better as an internal method?

Reply:
Sorry, meant like this
```golang
type DataField interface {
	getField() zap.Field
}
```
I think private interface methods are still compatible across versions, and this prevents unnecessary exposure of the zap abstraction.

Reply:
apparently you can